extension/package.json

```diff
@@ -45,39 +39,35 @@
   },
   "homepage": "https://github.com/commercetools/commercetools-adyen-integration#readme",
   "devDependencies": {
-    "adyen-cse-js": "Adyen/adyen-cse-web#0_1_23",
     "babel-eslint": "^10.1.0",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
-    "cypress": "^4.2.0",
-    "doctoc": "^1.4.0",
     "eslint": "^5.16.0",
     "eslint-config-airbnb": "^17.1.1",
     "eslint-config-commercetools": "^6.0.0",
-    "eslint-plugin-cypress": "^2.10.3",
-    "eslint-plugin-import": "^2.20.1",
+    "eslint-plugin-import": "^2.21.2",
     "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-react": "^7.19.0",
-    "mocha": "^7.1.0",
+    "eslint-plugin-react": "^7.20.0",
+    "mocha": "^8.0.1",
     "ngrok": "^3.2.7",
+    "nyc": "^15.1.0",
     "nock": "^12.0.3",
-    "nyc": "^15.0.0",
     "proxyquire": "^2.1.3",
-    "sinon": "^9.0.1",
-    "start-server-and-test": "1.10.11"
+    "sinon": "^9.0.2",
+    "start-server-and-test": "1.11.0"
```

Comment:
I think you can remove this dep. I don't see it's being used anywhere and before it was because of cypress with command `server-test`.